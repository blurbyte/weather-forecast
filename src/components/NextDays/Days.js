import styled from 'styled-components';

const Days = styled.ul`
  list-style: none;
  padding: 0;
  margin: 0;
  display: grid;
  grid-template-columns: repeat(4, 1fr);
`;

export default Days;
