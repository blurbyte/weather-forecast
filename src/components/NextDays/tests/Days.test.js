import React from 'react';
import renderer from 'react-test-renderer';

import Days from '../Days';

jest.mock('../../WeatherIcon', () => 'WeatherIcon');

test('renders correctly', () => {
  const days = {
    '5/21/2018': {
      weatherId: 800,
      description: 'clear sky',
      tempMax: 18.75,
      tempMin: 9.7
    },
    '6/21/2018': {
      weatherId: 800,
      description: 'clear sky',
      tempMax: 14.3,
      tempMin: 11.2
    },
    '7/21/2018': {
      weatherId: 801,
      description: 'clouds',
      tempMax: 11.23,
      tempMin: 9
    }
  };

  const component = renderer.create(<Days days={days} />);
  expect(component.toJSON()).toMatchSnapshot();
});
