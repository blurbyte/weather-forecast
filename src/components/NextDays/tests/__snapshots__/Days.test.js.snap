// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<ul
  className="sc-bdVaJa eqmkZd"
>
  <li
    className="sc-htpNat eywmEU"
  >
    <p
      className="sc-bwzfXH iezheg"
    >
      Monday
    </p>
    <div
      className="sc-bxivhb clTIXs"
    >
      <WeatherIcon
        small={true}
        weatherId={800}
      />
      <div
        className="sc-ifAKCX hNdbxq"
      >
        <p
          className="sc-EHOje dbfoHw sc-bwzfXH iezheg"
        >
          19
        </p>
        <p
          className="sc-EHOje dbfoHw sc-bwzfXH bkHTLu"
        >
          10
        </p>
      </div>
    </div>
    <p
      className="sc-bZQynM bPaBwE sc-bwzfXH dcdldk"
    >
      clear sky
    </p>
  </li>
  <li
    className="sc-htpNat eywmEU"
  >
    <p
      className="sc-bwzfXH iezheg"
    >
      Thursday
    </p>
    <div
      className="sc-bxivhb clTIXs"
    >
      <WeatherIcon
        small={true}
        weatherId={800}
      />
      <div
        className="sc-ifAKCX hNdbxq"
      >
        <p
          className="sc-EHOje dbfoHw sc-bwzfXH iezheg"
        >
          14
        </p>
        <p
          className="sc-EHOje dbfoHw sc-bwzfXH bkHTLu"
        >
          11
        </p>
      </div>
    </div>
    <p
      className="sc-bZQynM bPaBwE sc-bwzfXH dcdldk"
    >
      clear sky
    </p>
  </li>
  <li
    className="sc-htpNat eywmEU"
  >
    <p
      className="sc-bwzfXH iezheg"
    >
      Saturday
    </p>
    <div
      className="sc-bxivhb clTIXs"
    >
      <WeatherIcon
        small={true}
        weatherId={801}
      />
      <div
        className="sc-ifAKCX hNdbxq"
      >
        <p
          className="sc-EHOje dbfoHw sc-bwzfXH iezheg"
        >
          11
        </p>
        <p
          className="sc-EHOje dbfoHw sc-bwzfXH bkHTLu"
        >
          9
        </p>
      </div>
    </div>
    <p
      className="sc-bZQynM bPaBwE sc-bwzfXH dcdldk"
    >
      clouds
    </p>
  </li>
</ul>
`;
